pkg/gofr/cmd.go

```diff
@@ -121,6 +124,8 @@ func AddHelp(helperString string) Options {
 	}
 }
 
+// addRoute adds a new route to cmd's list of routes.
+// Optional configuration for the route is passed using functional options like AddHelp and AddDescription.
```

Comment:
`addRoute adds a new route to cmd's list of routes.` having this is sufficient

Reply:
Ok sir have addressed this in the new commit.